pytensor/tensor/basic.py

```diff
@@ -4269,6 +4269,25 @@ def take_along_axis(arr, indices, axis=0):
     return arr[_make_along_axis_idx(arr.shape, indices, axis)]
 
 
+def ix_(*args):
```

Comment:
Want to add this to `__all__`?

Reply:
I wanted to check with you first

Reply:
Doesn't really matter. It shouldn't be too hard right? It would be implemented as classmethod of `ix_`?

Reply:
Sorry I got confused, `np.ix_` is a normal function, so we can keep this as-is. If we wanted to do the other ones (`np.r_`, `np.c_`, `np.s_`, `np.ogrid`, `np.mgrid`), they have the funny syntax because they allow fast definition using slices (`np.ogrid[1:2, 1:2]` is the same as `np.ix_([1, 2], [1,2])`.

All of these live in [numpy.lib._index_tricks_impl.py](https://github.com/numpy/numpy/blob/9ab2a257e4f31f3524377d176233d0d4561c8fd4/numpy/lib/_index_tricks_impl.py).

So what I'm trying to say is I have no idea where to put `ix_`.

Reply:
Can be here no problem. We might refactor basic another time